crates/radix-engine-toolkit/src/transaction_types/traverser/types/general_subintent.rs

```diff
@@ -88,15 +88,15 @@ impl StaticAnalysisCallback for GeneralSubintentDetector {
             | InstructionV2::DropAllProofs(..)
             | InstructionV2::CallFunction(..)
             | InstructionV2::YieldToParent(_)
-            | InstructionV2::YieldToChild(_)
             | InstructionV2::VerifyParent(_) => true,
             /* Not Permitted */
             InstructionV2::BurnResource(..)
             | InstructionV2::CallRoyaltyMethod(..)
             | InstructionV2::CallMetadataMethod(..)
             | InstructionV2::CallRoleAssignmentMethod(..)
             | InstructionV2::CallDirectVaultMethod(..)
-            | InstructionV2::AllocateGlobalAddress(..) => false,
+            | InstructionV2::AllocateGlobalAddress(..)
+            | InstructionV2::YieldToChild(_) => false,
```

Comment:
Can we add a test for this? i.e. that a subintent yielding to CHILD is not matched?

Reply:
Similarly, given it's a requirement for the Gateway to return the correct classification for the MVP, can we have a test where a simple transaction intent manifest which has a YIELD_TO_CHILD doesn't match against anything? (with a comment that this behaviour is needed for the wallet to mark transactions-with-subintents as Complex; and if changed, we should ensure we change it in line with Gateway and/or wallet behaviour)